dcicutils/ff_utils.py

```diff
@@ -22,6 +22,8 @@
 HIGLASS_BUCKETS = ['elasticbeanstalk-fourfront-webprod-wfoutput',
                    'elasticbeanstalk-fourfront-webdev-wfoutput']
 
+PRODUCTION_ENVS = ['fourfront-blue', 'fourfront-green']
```

Comment:
We did this in `fourfront` in `commands/create_mapping_on_deploy.py` and really need to centralize these. I started a PR for that and will try to follow up later. It's bad to have these in multiple places since if we update one and not another we'll get a mess. I made a ticket C4-92. Please add a TODO comment citing that ticket.